src/ingestion/sources/base.py

```diff
@@ -246,7 +237,7 @@ def all(cls):
     @classmethod
     def default_for(cls, data_type): 
         data_type = data_type.lower()
-        if data_type == 'number':
+        if 'num' in data_type or 'int' in data_type:
```

Comment:
we should consider supporting other numeric types e.g. 

```
numeric_data_types = ['num', 'int', 'decimal', 'double', 'real', 'serial']

if any(substring in data_type for substring in numeric_data_types):
    ...

```

Reply:
Correct